### ISSUE TITLE
deprecated api in use

### ISSUE THREAD

**Issue by toddfries** [ironcamel/WebService-Coinbase#1]
There was the old api (which this uses), then there was the v1 api, then there is now the v2 api using oauth2.

Everything other than v2 api is deprecated.

Any chance we can see an update for v2 api?  Are there other perl modules that implement v2 api with coinbase?

**Comment by ironcamel** [comment]
Yeah, I can update this for v2. I am not aware of other coinbase modules. You can check https://metacpan.org
